lib/miq_automation_engine/engine/miq_ae_engine/drb_remote_invoker.rb

```diff
@@ -67,21 +67,23 @@ def drb_cache_timeout
 
     # code building
 
-    def build_method_content(body, method_name, miq_ae_service_token)
+    def build_method_content(bodies, method_name, miq_ae_service_token, line_hash)
       [
-        dynamic_preamble(method_name, miq_ae_service_token),
+        dynamic_preamble(method_name, miq_ae_service_token, line_hash),
         RUBY_METHOD_PREAMBLE,
-        body,
+        bodies.flatten,
         RUBY_METHOD_POSTSCRIPT
       ].join("\n")
```

Comment:
I feel like this should be changed to [].flatten.join("\n") instead of the flatten *inside* the array.  I know it works both ways, but I feel it's more intuitive this way.